cfme/tests/containers/test_chargeback_for_images.py

```diff
@@ -83,33 +59,607 @@ def assign_compute_custom_rate(new_chargeback_fixed_rate, provider):
+        soft_assert):
+    abstract_test_chargeback_cost(chargeback_report_for_weekly_variable_rate,
+                                  new_chargeback_weekly_variable_compute_rate,
+                                  'Network', 'Weekly', soft_assert)
+
+
+@pytest.mark.skip('This test is skipped due to a framework issue: '
+                  'https://github.com/ManageIQ/integration_tests/issues/5027')
+@pytest.mark.long_running_env
+@pytest.mark.polarion('CMP-<TBD>')
+def test_image_chargeback_cpu_cores_used_weekly_variable_rate(
+        chargeback_report_for_weekly_variable_rate, new_chargeback_weekly_variable_compute_rate,
+        soft_assert):
+    abstract_test_chargeback_cost(chargeback_report_for_weekly_variable_rate,
+                                  new_chargeback_weekly_variable_compute_rate,
+                                  'CpuCores', 'Weekly', soft_assert)
+
+
+@pytest.mark.skip('This test is skipped due to a framework issue: '
+                  'https://github.com/ManageIQ/integration_tests/issues/5027')
+@pytest.mark.long_running_env
+@pytest.mark.polarion('CMP-<TBD>')
+def test_image_chargeback_memory_used_monthly_variable_rate(
+        chargeback_report_for_monthly_variable_rate, new_chargeback_monthly_variable_compute_rate,
+        soft_assert):
+    abstract_test_chargeback_cost(
+        chargeback_report_for_monthly_variable_rate, new_chargeback_monthly_variable_compute_rate,
+        'Memory', 'Monthly', soft_assert)
+
+
+@pytest.mark.skip('This test is skipped due to a framework issue: '
+                  'https://github.com/ManageIQ/integration_tests/issues/5027')
+@pytest.mark.long_running_env
+@pytest.mark.polarion('CMP-<TBD>')
+def test_image_chargeback_network_used_monthly_variable_rate(
+        chargeback_report_for_monthly_variable_rate, new_chargeback_monthly_variable_compute_rate,
+        soft_assert):
+    abstract_test_chargeback_cost(chargeback_report_for_monthly_variable_rate,
+                                  new_chargeback_monthly_variable_compute_rate,
+                                  'Network', 'Monthly', soft_assert)
+
+
+@pytest.mark.skip('This test is skipped due to a framework issue: '
+                  'https://github.com/ManageIQ/integration_tests/issues/5027')
 @pytest.mark.long_running_env
-@pytest.mark.long_running_provider
-@pytest.mark.polarion('CMP-10432')
-def test_image_chargeback_fixed_rate(chargeback_report_custom):
-    assert chargeback_report_custom, 'Error in produced report, No records found'
+@pytest.mark.polarion('CMP-<TBD>')
+def test_image_chargeback_cpu_cores_used_monthly_variable_rate(
+        chargeback_report_for_monthly_variable_rate, new_chargeback_monthly_variable_compute_rate,
+        soft_assert):
+    abstract_test_chargeback_cost(chargeback_report_for_monthly_variable_rate,
+                                  new_chargeback_monthly_variable_compute_rate,
+                                  'CpuCores', 'Monthly', soft_assert)
```

Comment:
Same with all these other tests. They should be parametrized, not copy pasted like this...